python/triton/language/math.py

```diff
@@ -157,7 +157,7 @@ def sqrt(x, _builder=None):
 
 @core.builtin
 @_check_dtype(dtypes=["fp32"])
-@_add_math_1arg_docstr("precise square root (rounding to nearest)")
+@_add_math_1arg_docstr("precise square root (rounding to nearest wrt the IEEE standard)")
```

Comment:
what is the motivation for changing this? Was the phrasing ambiguous?

Reply:
People (not familiar with IEEE standard) may get confused about the meaning of "precise" vs "rounding"... 
This change explicitly highlights that the rounding operation adheres to a well-defined standard.
Thanks